DEBIAN/postrm

```diff
@@ -11,6 +12,16 @@ case "$1" in
                 db_purge
         fi
         rm -f /usr/sbin/kiosk-browser
+	# In a perfect world, this should also care for lightdm xdm wdm slim lxdm gdm etc...
+	WMS=`dpkg-query -W -f='Package: ${Package}\nProvides: ${Provides}\n'   | grep -B 1 -E "^Provides: .*x-display-manager" | grep Package | cut -d" " -f2`
```

Comment:
If the user breaks it he will fix it. I really don't like this lookup into
tha package database for this purpose. I also think that this is less
likely scenario and that at the most a user will just remove kiosk-browser
and be happy to go back to the previous configuration.

On 13 October 2015 at 14:03, MartinLinSup <notifications@github.com> wrote:

> In DEBIAN/postrm
> <https://github.com/ImmobilienScout24/kiosk-browser/pull/14#discussion_r41857053>
> :
>
> > @@ -11,6 +12,16 @@ case "$1" in
> >                  db_purge
> >          fi
> >          rm -f /usr/sbin/kiosk-browser
> > +	# In a perfect world, this should also care for lightdm xdm wdm slim lxdm gdm etc...
> > +	WMS=`dpkg-query -W -f='Package: ${Package}\nProvides: ${Provides}\n'   | grep -B 1 -E "^Provides: .*x-display-manager" | grep Package | cut -d" " -f2`
>
> The user could have removed a DM which was configured at installtime.
> Remebering this would not lead to a working DM.
>
> Will change `` to $()
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ImmobilienScout24/kiosk-browser/pull/14/files#r41857053>
> .
>